protocol/x/clob/keeper/authenticators.go

```diff
@@ -0,0 +1,73 @@
+package keeper
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	accountpluslib "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/lib"
+	aptypes "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// MaybeValidateAuthenticators checks if the transaction has authenticators specified and if so,
+// validates them. It returns an error if the authenticators are not valid or removed from state.
+func (k Keeper) MaybeValidateAuthenticators(ctx sdk.Context, txBytes []byte) error {
```

Comment:
None of the logic here is specific to x/clob - should this function be a method in CLOB or Accountplus?